internal.lua

```diff
@@ -18,6 +18,23 @@ function areas:save()
 	file:close()
 end
 
+local function populateStore(self)
+	if not AreaStore then
+		return
+	end
+	local store = AreaStore()
+	local store_ids = {}
+	store:reserve(#self.areas)
+	for id, area in pairs(areas.areas) do
+		local sid = store:insert_area(area.pos1,
+			area.pos2, dump(id))
```

Comment:
`dump()` is for display, it's format is not designed to be read by anything other than a human and its output may not be consistent.  Use `tostring` here instead, it's also faster.